cantools/database/can/formats/arxml/system_loader.py

```diff
@@ -1478,49 +1476,40 @@ def _load_signal(self, i_signal_to_i_pdu_mapping):
         # loading initial values is way too complicated, so it is the
         # job of a separate method
         initial = self._load_arxml_init_value_string(i_signal, system_signal)
```

Comment:
that should be `raw_initial`?!

Reply:
No, this is the string value, i'll rename the variable